site/book/04-using-functions/01-declarative-function-execution.md

```diff
@@ -66,12 +66,12 @@ Now, let's render the package hierarchy:
 $ kpt fn render wordpress
 Package "wordpress/mysql":
 
-[PASS] "gcr.io/kpt-fn/set-label:v0.1"
+[PASS] "set-label:v0.1"
```

Comment:
Output should show the fully expanded image name.

Reply:
updated.